templates/boundary/abstract.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+{% block content %}
+<h3>Boundary data</h3>
+<p>These data are intended for exposing the physical, administrative, and political polygon boundaries used across SNAP tools and data.</p>
```

Comment:
I think somewhere here we want to say what coordinate system the data are in. Would even be a neat feature to allow specifying the coordinate system to download in and shouldn't be that hard with a simple `to_crs()` call on the geodataframe. See forthcoming comment on PR about choosing a common default CRS.